cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -907,6 +907,66 @@ ALTER DATABASE ... SET TOPOLOGY n PRIMAR{Y\|IES} [m SECONDAR{Y\|IES}]
 a|
 New sub-clause for `ALTER DATABASE`, which allows modifying the number of servers hosting a database in cluster environments.
 
+a|
+label:functionality[]
+label:added[]
+[source, cypher, role="noheader"]
+----
+ENABLE SERVER ...
+----
+a|
+New Cypher command for enabling servers.
+
+a|
+label:functionality[]
+label:added[]
+[source, cypher, role="noheader"]
+----
+ALTER SERVER SET OPTIONS
+----
+a|
+New Cypher command for setting options for a server.
+
+a|
+label:functionality[]
+label:added[]
+[source, cypher, role="noheader"]
+----
+RENAME SERVER ... TO ...
+----
+a|
+New Cypher command for changing the name of a server.
+
+a|
+label:functionality[]
+label:added[]
+[source, cypher, role="noheader"]
+----
+REALLOCATE DATABASES
+----
+a|
+New Cypher command for re-balancing what servers host which databases.
+
+a|
+label:functionality[]
+label:added[]
+[source, cypher, role="noheader"]
+----
+DEALLOCATE DATABASES FROM SERVERS ...
```

Comment:
Is it always _SERVERS_ (in plural)?

Reply:
no, but do we have to list the complete syntax in additions?

Reply:
Would it make sense to write

`DEALLOCATE DATABASES FROM SERVER[S]` like in the CIP?